src/execution_plan/execution_plan_build/execution_plan_construct.c

```diff
@@ -206,8 +206,8 @@ static void _buildForeachOp
 
```

Comment:
Tried this, But there is a problem with it since we may not want to clone the ArgumentList operation when cloning Foreach. Also, I'm not sure we want have access to the wanted ArgumentList op (is it always the same existing one?).